trigger/process_test.go

```diff
@@ -626,6 +626,11 @@ func TestProjectTriggerIntegrationForPush(t *testing.T) {
 	require.NotNil(mani)
 	assert.Equal(downstreamProjectRef.Id, mani.ProjectName)
 	assert.Equal(uptreamProjectRef.Branch, mani.Branch)
+	assert.Len(mani.Modules, 1)
+	assert.Equal("3585388b1591dfca47ac26a5b9a564ec8f138a5e", mani.Modules["sample"].Revision)
```

Comment:
To avoid nil pointer panicking, you would have to `require.NotNil` for `mani.Modules["sample"]` before dereferencing `Revision`.